proto/hex_pb_names.proto

```diff
@@ -13,4 +13,7 @@ message Package {
 
   // If set, the name of the package repository (NEVER USED, DEPRECATED)
   // string repository = 2;
+
+  // unix timestamp
+  optional fixed32 updated_at = 3;
```

Comment:
Can we use this type https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp or at least something more accurate with sub-second resolution?

Reply:
Sounds good, see: https://github.com/hexpm/hex_core/pull/114/files#diff-61531a44d8f89a5934e78d85759a628de1a951d06c03b74ccbc7036dada837a0R22